### PR TITLE
Reactivate static build tests. 

### PR DESCRIPTION
**Pull request by daniellowell**
Reverts changes from [this commit](https://github.com/ROCmSoftwarePlatform/MIOpen/commit/441bb75eabc2d0eeff42fbfc72b4e961693a662b).

Added Aux parallel stage.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -229,76 +229,88 @@ pipeline {
                     }
                 }
 
-                /*stage('gfx908 GCC Debug') {
-                    agent{ label rocmnode("gfx908") }
-                    steps{
-                        buildJob('g++-5', '-DMIOPEN_TEST_GFX908=On -DBUILD_DEV=On -DCMAKE_BUILD_TYPE=debug', "", image, "")
-                    }
-                }*/
```
Comment (pfultz2) [inline Jenkinsfile:237]: Why is this commented it out?

**Comment by atamazov** [comment]
> Reverts changes from [this commit](https://github.com/ROCmSoftwarePlatform/MIOpen/commit/441bb75eabc2d0eeff42fbfc72b4e961693a662b).

It doesn't revert 441bb75eab. Do you mean some other commit?

File: Jenkinsfile (hunk 1 above)
Comment (daniellowell) [reply Jenkinsfile:237]: Dunno it was commented out at some point. I just deleted it.

**Comment by daniellowell** [comment]
Whoops, sorry @atamazov looks like this was "On" from the #137 
[Reference comment.](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/137/files#r439518280)

**Comment by atamazov** [comment]
PR description fixed.

**Pull request merged**